nomic-bot/src/universe/commands/happy.js

```diff
@@ -0,0 +1,24 @@
+import { addCommand, updatePlayer, createComment } from './index';
+import moment from 'moment';
+import { chain, startsWith } from 'lodash';
+
+
+addCommand('/happy', (_, { playerComments, player }) => {
+    const now = moment();
+    const latestHappyCommentWeeks = chain(playerComments)
+        .filter(comment => startsWith(comment.body, '/happy'))
+        .map(comment => now.diff(moment(comment.created_at), 'week'))
+        .min()
+        .value()
+
+    if (latestHappyCommentWeeks === undefined || latestHappyCommentWeeks >= 1) {
+        player.village.happiness += 1;
```

Comment:
We have a law that all active players must have a village, but we have no controls in place to prevent committing a players.yml file with an active player and no village.